src/Binance/CoinMFutures/CoinMFutures.jl

```diff
@@ -20,4 +23,13 @@ using .IncomeLog
 include("API/OrderBook.jl")
 using .OrderBook
 
+include("API/premiumIndex.jl")
+using .PremiumIndex
+
+include("API/FundingRate.jl")
+using .FundingRate
+
+include("API/Ticker24hr.jl")
+using .Ticker24hr
```

Comment:
It would be great to arrange them in alphabetical order

Reply:
Something like this

```
include("API/Candle.jl")
using .Candle

include("API/ContinuousCandle.jl")
using .ConitinuousCandle

include("API/ExchangeInfo.jl")
using .ExchangeInfo

include("API/FundingRate.jl")
using .FundingRate

include("API/IncomeLog.jl")
using .IncomeLog

include("API/OrderBook.jl")
using .OrderBook

include("API/PremiumIndex.jl")
using .PremiumIndex

include("API/Ticker.jl")
using .Ticker
```